public/js/main.js

```diff
@@ -228,10 +227,23 @@ bingo.controller('bingoController', function($scope, $document, $http, $routePar
         console.log(data);
         $scope.gamecard = data.card.squares;
         $scope.cardid = data.card._id;
-        $scope.roomname = data.game.room;
-        $scope.currentUser = data.user;
-        // $scope.host_name = data.game.host.name;
-        $scope.players = [];
+
+        $scope.roomname = data.roomname;
```

Comment:
Again, please be careful when merging. This also undid changes introduced in the [last pull request](https://github.com/RyanEggert/olinjs-bingo/pull/27/files#diff-f485b1889bd3c96211c9236b0cf5daecR206). 
It seems that `.roomname` doesn't exist; `.game.room` is what we want (and what it was).